benefits/static/css/styles.css

```diff
@@ -696,8 +694,8 @@ a.card:focus-visible {
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25) !important;
 }
 
-.card-row {
-  gap: var(--card-gap);
+.agency-container-width {
+  width: 75%;
 }
```

Comment:
Thanks @machikoyasuda!

Initially I was having trouble getting all the layouts to show up correctly using `width: 75%;` only so I tried out `grid`, but with 
```
width: 75%;
margin: 0 auto;
```
as you wrote in the ticket makes all the layouts work. I'll update this PR shortly to keep using `flex` instead of `grid`.